sdk/src/main/java/co/omise/android/models/Source.kt

```diff
@@ -79,7 +79,7 @@ data class Source(
         @JsonProperty("phone_number")
         private var phoneNumber: String? = null
         @JsonProperty("installment_term")
-        private var installmentTerm: Int = 0
+        private var installmentTerm: Int? = null
```

Comment:
Cause it can be null from our API right?

Reply:
Yup.